public/client/src/components/RatingsAndRevs/ratingBreakdown.jsx

```diff
@@ -35,7 +35,7 @@ class RatingBreakdown extends React.Component {
     return (
       <div className="rating-breakdown">
         <h3>Ratings and Reviews</h3>
-        <Stars average={avgRating} size={25}/>updateFilters
+        <Stars average={avgRating} size={25}/>
         <p>{avgRating} Stars, {totalRatings} Ratings</p>
         <IndividualRatingBreakdown numStars={5} percent={numFiveStars} ratings={ratings[5]} updateFilters={this.props.updateFilters}/>
```

Comment:
potential refactor for dryer code: 
create for loop : 
`for (let i =0; i < 6; i++)`
create component for each iteration of i
`<IndvidualRatingBreakdown numStars={i} percent={num${i}Stars} ratings={ratings[i]}...`